src/main.rs

```diff
@@ -112,8 +114,13 @@ fn main() -> Result<(), Error> {
     );
     headers.insert(CONTENT_TYPE, "application/json".parse().unwrap());
     let json_data = serde_json::to_string(&data)?;
+    let timeout_secs = env::var("CHATGPT_REQUEST_TIMEOUT_SECS")
```

Comment:
Let's name this as `CHATGPT_CLI_REQUEST_TIMEOUT_SECS`. This way we can start following a `CHATGPT_CLI_*` pattern for environment variables